### PR TITLE
Fix broken links and port remaining docs

### PR DESCRIPTION
**Pull request by ckarenz**
This ports all remaining documentation from `allenai/beaker`.

### CODE REVIEW

File: docs/start/install.md [hunk 1]
```diff
@@ -14,17 +14,21 @@ If you already have an account, make sure to sign in before using Beaker.
...
-[release](https://github.com/allenai/beaker/releases) and extract it:
+- On Linux, download the latest [release](https://github.com/allenai/beaker/releases) and extract it.
 
     ```bash
-    tar -xvzf beaker_*.tar.gz -C /usr/local/bin
+    curl -s https://api.github.com/repos/allenai/beaker/releases/latest \
+      | grep "beaker_linux.tar.gz" \
+      | cut -d : -f 2,3 \
+      | tr -d \" \
+      | wget -qi -
```
Comment (codeviking) [inline docs/start/install.md:31]: Don't we still need to decompress it? Maybe `| tar -xvzf beaker_*.tar.gz -C /usr/local/bin`?

File: docs/start/install.md [hunk 2]
```diff
@@ -36,22 +40,26 @@ There are a few different ways to install Beaker:
     brew install beaker
     ```
 
-- [GoLang](https://golang.org/) developers can install Beaker from source using standard tools:
+- You can also install Beaker directly from source using [Go](https://golang.org/) standard tools.
 
     ```bash
-    go get -u github.com/allenai/beaker/...
+    go install github.com/allenai/beaker/cmd/beaker@latest
```
Comment (codeviking) [inline docs/start/install.md:46]: 👍

File: docs/start/interactive.md [hunk 3]
```diff
@@ -270,7 +270,7 @@ The session will stop when the main process exits.
 You can also provide a command to `exec`. This example prints GPU utilization:
 
 ```
-beaker session exec <session> nvidia-smi
+beaker session exec <session> -- nvidia-smi
```
Comment (codeviking) [inline docs/start/interactive.md:273]: 👍

File: docs/start/selfhost.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
+# Bring Your Own Compute
+
+Beaker supports hosting your own compute resources. You can manage a single machine (node) or
+manage many nodes under one or more clusters.
+
+## Pre-requisites
+
+1. Only Linux operating systems with systemd are supported.
+1. Docker must be installed.
```
Comment (codeviking) [inline docs/start/selfhost.md:9]: Do we know what the minimum version is that we support? Ideally that'd be here too.

File: docs/start/selfhost.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+   ```
+
+   The account may be an individual account or an organization. It should be the same account under
+   which you plan to run experiments. Clusters may be used by any member of their associated account.
+
+   \* Members of the `ai2` organization may choose from among our
+   [existing clusters](https://beaker.org/clusters).
+
+1. Beaker requires an agent (executor) to manage jobs on each node. The executor coordinates with
+   [beaker.org](https://beaker.org/clusters), startst jobs, and monitors their execution. The
```
Comment (codeviking) [inline docs/start/selfhost.md:31]:
```suggestion
   [beaker.org](https://beaker.org/clusters), starts jobs, and monitors their execution. The
```

File: docs/start/selfhost.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+
+   ```
+   beaker cluster create <account>/<name>
+   ```
+
+   The account may be an individual account or an organization. It should be the same account under
+   which you plan to run experiments. Clusters may be used by any member of their associated account.
+
+   \* Members of the `ai2` organization may choose from among our
+   [existing clusters](https://beaker.org/clusters).
```
Comment (codeviking) [inline docs/start/selfhost.md:28]: This to me implies that users might decide to register their host as a member of an AI2 cluster. Is that specifically what we're trying to encourage here, or should users always create a new one?

File: docs/start/selfhost.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+
+1. Beaker requires an agent (executor) to manage jobs on each node. The executor coordinates with
+   [beaker.org](https://beaker.org/clusters), startst jobs, and monitors their execution. The
+   executor is also responsible for capturing and uploading results.
+
+   Beaker offers a command to install and configure the executor as a systemd service.
+   
+   ```bash
+   sudo beaker executor install <account/cluster>
+   ```
```
Comment (codeviking) [inline docs/start/selfhost.md:38]: TIL. This is cool!

**Review by codeviking — APPROVED** [review]
LGTM! The docs for self-hosting are a great addition!

File: docs/start/install.md (hunk 1 above)
Comment (ckarenz) [reply docs/start/install.md:32]: Yes.  I had this in my editor but forgot to save.  Fixed now.

File: docs/start/selfhost.md (hunk 4 above)
Comment (ckarenz) [reply docs/start/selfhost.md:9]: We don't.  It's a pretty wide range so I don't anticipate most installers would have an issue here.  If they do the CLI will produce a reasonably clear error.

File: docs/start/selfhost.md (hunk 6 above)
Comment (ckarenz) [reply docs/start/selfhost.md:28]: I'm going to remove the ai2-specific instructions.  This should work for anyone.

**Comment by ckarenz** [comment]
> The docs for self-hosting are a great addition!

Lane did most of the work on these.  I'm just polishing and publishing. :)

**Pull request merged**